### PR TITLE
ci(gitlab): update to SocialGouv/gitlab-ci-yml#autodevops 12.0.0

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by revolunet** [comment]
shouldn't the env use `gitlab-ci.variables.PORT` ?

File: .k8s.app.values.yml [hunk 1]
```diff
@@ -15,5 +15,11 @@ deployment:
     requests:
       memory: 64Mi
 
+  env:
+    - name: PORT
+      value: "4000"
```
Comment (douglasduteil) [inline .k8s.app.values.yml:20]: @revolunet here ?

Comment (douglasduteil) [inline .k8s.app.values.yml:20]:
```suggestion
      value: "${PORT}"
```

Comment (revolunet) [reply .k8s.app.values.yml:20]: Yes?

Comment (douglasduteil) [reply .k8s.app.values.yml]: done :)

**Pull request merged**